tests/SbrpTests/ValidationTests.cs

```diff
@@ -0,0 +1,172 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+using NuGet.Packaging.Signing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SbrpTests;
+
+public class ValidationTests
+{
+    private const string SbrpAttributeType = "System.Reflection.AssemblyMetadataAttribute";
+    private const string VersionPattern = @"(\.\d)+([\-\w])*";
+    private string[] Packages = Array.Empty<string>();
+    public ITestOutputHelper Output { get; set; }
+
+    public ValidationTests(ITestOutputHelper output) => Output = output;
+
+    [SkippableFact]
+    public async void ValidateSbrp()
+    {
+        Skip.If(RuntimeInformation.IsOSPlatform(OSPlatform.Windows), "This test is not supported on Windows.");
+        
+        InitializeValidationTests();
+
+        CheckForSbrpAttribute();
```

Comment:
I wanted to skip the whole test class if the test is running on windows. This means skipping the constructor and both tests in the class. But xunit does not have this feature. So I had two choices: either combine the constructor (now InitializeValidationTests) and the two tests into one test and skip that one test, or repeat the constructor’s work in each test and remove the shared members. I chose the first option because it seemed neater. However, I realize that this makes the test results less visible.

Reply:
This solution doesn't scale well at least for now.  If we want to add additional pieces to validate, then it implies is all has to be in the same testcase.  We also may want additional validation that differs per package type.  Putting all logic in a single test case smells.

Reply:
That makes sense. I've refactored into two test cases in https://github.com/dotnet/source-build-reference-packages/pull/817/commits/2aae3ceba8c63acabf1af5acaeded0330785d706